<launch>
	<include file="$(find openni_launch)/launch/openni.launch" />
	<node name="four95_pcl_node" pkg="495_pcl" type="495_pcl_node" />
	<node name="rviz" pkg="rviz" type="rviz" />
</launch>
